BusyTiming/ViewControllers/TimerViewController.swift

```diff
@@ -21,9 +38,18 @@ class TimerViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        timerVM = timerModel.timerVM
-        timerVM.timerDelegate = self
-        timerModel.timerOn()
+        timerStart()
+    }
+    
+    func timerStart () {
```

Comment:
all the timer-related logic should be in the view model. view model should expose a publisher of string values for the timer label. thw view should just bind this publisher to the label.